beacon-light-client/solidity/hardhat.config.ts

```diff
@@ -58,15 +64,61 @@ export default {
       url: `https://ropsten.infura.io/v3/${mandatoryConf.INFURA_API_KEY}`,
       accounts: [optionalConf.USER_PRIVATE_KEY],
     },
+    sepolia: {
+      url: `https://eth-sepolia.g.alchemy.com/v2/${optionalConf.ALCHEMY_API_KEY}`,
+      accounts: [optionalConf.USER_PRIVATE_KEY],
+    },
     goerli: {
-      url: `https://goerli.infura.io/v3/${mandatoryConf.INFURA_API_KEY}`,
+      url: `https://eth-goerli.g.alchemy.com/v2/${optionalConf.ALCHEMY_API_KEY}`,
+      accounts: [optionalConf.USER_PRIVATE_KEY],
+    },
+    optimisticGoerli: {
+      url: `https://opt-goerli.g.alchemy.com/v2/${optionalConf.ALCHEMY_API_KEY}`,
+      accounts: [optionalConf.USER_PRIVATE_KEY],
+    },
+    baseGoerli: {
+      url: 'https://base-goerli.rpc.thirdweb.com',
+      accounts: [optionalConf.USER_PRIVATE_KEY],
+    },
+    arbitrumGoerli: {
+      url: `https://arb-goerli.g.alchemy.com/v2/${optionalConf.ALCHEMY_API_KEY}`,
+      accounts: [optionalConf.USER_PRIVATE_KEY],
+      contractAddress: '0xB94868ba0903883bD2dE3311Fc377f3c50D602eA',
+    },
+    mumbai: {
+      url: `https://polygon-mumbai.g.alchemy.com/v2/${optionalConf.ALCHEMY_API_KEY}`,
       accounts: [optionalConf.USER_PRIVATE_KEY],
     },
   },
   mocha: {
     timeout: 100000000,
   },
   etherscan: {
-    apiKey: mandatoryConf.ETHERSCAN_API_KEY,
+    apiKey: {
+      optimisticGoerli: optionalConf.OPTIMISM_ETHERSCAN_API_KEY,
```

Comment:
The README has a section where all required ENV variables are described.
If these are now mandatory, please include them there as well.